MAPL_Profiler/TextColumn.F90

```diff
@@ -101,9 +101,10 @@ end subroutine set_separator
 
    ! Would be a function, but this is a workaround for gfortran 8.2
    ! issue with allocatable arrays of deferred length strings.
-   subroutine get_separator(this, separator)
+   subroutine get_separator(this, separator, k)
       class(TextColumn), intent(in) :: this
-      character(*), intent(inout) :: separator(:)
+      character(*), intent(inout) :: separator(k)
+      integer, intent(in) :: k
```

Comment:
Ooh. You might want to flip these around. I know I've used a compiler that didn't like it in this order. Aka:
```fortran
real :: x(n)
integer :: n
```
failed but:
```fortran
integer :: n
real :: x(n)
```
succeeded.

But maybe that doesn't matter anymore?

Reply:
If it is safer to flip, why not ? done !